bin/dota/objects/GodScrewdriver.java

```diff
@@ -0,0 +1,47 @@
+package dota.objects;
+
+import dota.logic.GameLogic;
+
+import java.util.ArrayList;
+
+public class GodScrewdriver extends Hero {
```

Comment:
Create separate folder for heroes, like /heroes and move all hero there, only basic class can be placed in root